kafo_parsers.gemspec

```diff
@@ -25,4 +25,5 @@ Gem::Specification.new do |spec|
   # puppet manifests parsing
   spec.add_development_dependency 'puppet', '< 4.0.0'
```

Comment:
This version spec could be changed now as Puppet 4.x + puppet-strings is suitable, perhaps removed entirely or set to `< 5`.